### ISSUE TITLE
Accessibility improvement

### ISSUE THREAD

**Issue by drxcheng** [Uniswap/interface#3396]
**Is your feature request related to a problem? Please describe.**
There are several accessibility issues on the interface by the WCAG standard (https://www.w3.org/WAI/standards-guidelines/wcag/)
1. no skip link, no level one heading (https://www.w3.org/TR/WCAG21/#bypass-blocks)
2. focus is not visible (https://www.w3.org/TR/WCAG21/#focus-visible)
3. lack of proper landmarks (https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA11)

**Describe the solution you'd like**
1. improve landmarks
2. add skip link and level one heading
3. improve focus styling

I'm happy to send pull request to fix step by step.
